spec/models/post_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] All the methods in the `Post` Model should be tested, kindly make sure to implement unit tests for the `update_posts_likes_counter` method